### PR TITLE
python-docx: init at 0.8.6

### PR DESCRIPTION
**Pull request by AJChapman**
###### Motivation for this change
This library is a required dependency for something I need to build.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/python-docx/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, lxml
+}:
+
+buildPythonPackage rec {
+  pname = "python-docx";
+  version = "0.8.6";
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/python-docx/default.nix]: The `name` attribute is added by `buildPython*` and should therefore be removed.

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by FRidh** [comment]
@GrahamcOfBorg build python3.pkgs.python-docx

**Comment by FRidh** [comment]
Test dependencies are missing.

**Comment by AJChapman** [comment]
I may have fixed these, not sure if my nix.useSandbox = true is working...

**Comment by AJChapman** [comment]
@GrahamcOfBorg build python3.pkgs.python-docx

**Comment by AJChapman** [comment]
Ok @FRidh, I think I have it working now.

File: pkgs/development/python-modules/python-docx/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+buildPythonPackage rec {
+  pname = "python-docx";
+  version = "0.8.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "117i84s6fcdsrfckbvznnqgqwhnf1x0523ps16cki8sg9byydv2m";
+  };
+
+  buildInputs = [ behave lxml pytest pyparsing mock ];
```
Comment (dotlambda) [inline pkgs/development/python-modules/python-docx/default.nix]: lxml belongs into `propagatedBuildInputs`, the rest into `checkInputs`

File: pkgs/development/python-modules/python-docx/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  propagatedBuildInputs = [ ];
+
+  meta = {
+    description = "Create and update Microsoft Word .docx files";
+    homepage = "https://python-docx.readthedocs.io/en/latest/";
```
Comment (dotlambda) [inline pkgs/development/python-modules/python-docx/default.nix]: no quotes

**Review by dotlambda — CHANGES_REQUESTED** [review]
Please squash the commits and use
```
pythonPackages.python-docx: init at 0.8.6
```
as the commit message.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python3.pkgs.python-docx

**Comment by dotlambda** [comment]
> Ran 0 tests in 0.000s

Please specify an appropriate `checkPhase`, e.g.
```
checkPhase = ''
  py.test
'';
```

**Comment by AJChapman** [comment]
checkPhase added, dependencies updated, commits squashed, and commit message updated. Thanks for the thorough review!

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.python-docx python3.pkgs.python-docx

**Pull request merged**